client/src/app/modules/error-handler/http-request-interceptor.ts

```diff
@@ -40,10 +40,7 @@ export class HttpRequestInterceptor implements HttpInterceptor {
 
     return next.handle(req).pipe(
       catchError((error: HttpErrorResponse) => {
-        if (error.status === 401) {
```

Comment:
I guess the purpose of this if statement was to differ between authentication related errors and any other kind of error. If the if statement is gone, a navigation change will always be triggered.

Reply:
Re-added the if clause.